src/Workspaces/Core/Portable/Workspace/Solution/ProjectState_Checksum.cs

```diff
@@ -27,6 +27,14 @@ public async Task<Checksum> GetChecksumAsync(CancellationToken cancellationToken
             return collection.Checksum;
         }
 
+        public Checksum GetParseOptionsChecksum(CancellationToken cancellationToken)
+            => GetParseOptionsChecksum(_solutionServices.Workspace.Services.GetService<ISerializerService>(), cancellationToken);
```

Comment:
could this be nullable enabled, and then GetRequiredService?

Reply:
i tried, but the impact was very large on this file.  and i don't like doing NRT on a file in piecemeal fashion.  Thanks!